client/landing/gutenboarding/components/confirm-purchase-modal/style.scss

```diff
@@ -0,0 +1,37 @@
+.confirm-purchase-modal__content {
+	padding: 32px;
+	max-width: 560px;
+}
+
+.confirm-purchase-modal__header {
+	font-family: 'Recoleta-Regular', 'Recoleta', serif;
+	font-size: 40px;
+	color: var( --studio-blue-40 );
+	margin: 0 0 16px;
+	text-align: center;
+}
+
+.confirm-purchase-modal__items {
+	font-size: 16px;
+	color: var( --studio-gray-60 );
+	font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial,
+		sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+	line-height: 1.5em;
+	padding: 32px;
+
+	ul {
+		list-style: circle;
+		margin-top: 16px;
+		margin-bottom: 32px;
+	}
+	li {
+		margin-left: 64px;
+	}
```

Comment:
Sooo waiting for a proper design system so that we wouldn't need to override basic UL lists CSS. :-D